content/v2.0/reference/cli/influx/apply/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: influx apply
+description: The 'influx apply' command applies InfluxDB templates.
+menu:
+  v2_0_ref:
+    name: influx apply
+    parent: influx
+weight: 101
+aliases:
+  - /v2.0/reference/cli/influx/pkg/
+v2.0/tags: [templates]
+---
+
+The `influx apply` command applies InfluxDB templates.
+_For information about finding and using InfluxDB templates, see
+[Use InfluxDB templates](/v2.0/influxdb-templates/use/)._
+
+## Usage
+```
+influx apply [flags]
+```
+
+#### Aliases
+`apply`, `pkg`
```

Comment:
would it be possible to mark the `pkg` alias as deprecated?

Reply:
Done.